src/DynamoCore/App.config

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <appSettings>
+   <configSections>
+      <section name="DebugModes" type="System.Configuration.AppSettingsSection"/>
+   </configSections>
+
```

Comment:
To add enable a debug mode (from config file)...one would have to add a new entry under a DebugModes section like so:
`
   <DebugModes>
      <add key="DebugMode1" value="true"/>
      <add key="DebugMode2" value="true"/>
   </DebugModes>
`

Reply:
I could add a separate config file...(debug.config) just so that we do not have to pollute the DynamoCore config ...